fusesoc/main.py

```diff
@@ -528,6 +540,17 @@ def get_parser():
     parser_core_show.add_argument("core", help="Name of the core to show")
     parser_core_show.set_defaults(func=core_info)
 
+    # tool subparser
+    parser_tool = subparsers.add_parser(
+        "tool", help="Subcommands for dealing with tools"
+    )
+    tool_subparsers = parser_tool.add_subparsers()
+    parser_tool.set_defaults(subparser=parser_tool)
+
+    # tool list subparser
+    parser_tool_list = tool_subparsers.add_parser("list", help="List available tools")
+    parser_tool_list.set_defaults(func=list_tools)
+
     # list-cores subparser
     parser_list_cores = subparsers.add_parser("list-cores", help="List available cores")
     parser_list_cores.set_defaults(func=list_cores)
```

Comment:
Looks like this command should then be "core list" as well in a follow-up?

Reply:
`list-cores` and `core-info`have been superseded by `core list` and `core show` since 8284202ed54e0ca0b6e57b6cddffa078b1aa4c9b Planning to remove the old commands at some points. Doing it at 2.0 would had been nice but we will need to do it later since we haven't put in a deprecation notice yet